scss/atoms/typography/_typography.scss

```diff
@@ -54,7 +54,7 @@ h6 {
 	font-size: em( 16px );
 	font-weight: 600;
 	line-height: 1;
-	text-transform: uppercase;
+	text-transform: uppercase; // FIXME Is this opinionated?
 }
```

Comment:
@sfrisk I don't think something like text-transform on a heading should be a variable.. we would have too many variables that way. h1-text-transform....to ...h6-text-transform. I think if we leave that property to default, users can overwrite with css easily?